### DISCUSSION TITLE
Question about distribution and software license.

### DISCUSSION THREAD

**Discussion by Im-Oab** [17cupsofcoffee/tetra#272] [General]
Is it ok If I use `cargo-about` and put the HTML file in the same folder as the executable file?

![Screen Shot 2021-07-01 at 5 17 46 pm](https://user-images.githubusercontent.com/622642/124082825-7ea7bd00-da90-11eb-9ab1-59d16a72c54a.png)

**Comment by 17cupsofcoffee** [comment]
_obligatory "i'm not a lawyer" disclaimer_

Yes, I think that's fine, and it matches what I've done with my own games in the past - as long as the licenses are provided in the distribution in some form, you should be good.

`std` is also Apache/MIT licensed, so you *technically* should include a license for that as well, since the code gets linked into your program. I don't think there's a nice automated way of doing that with `cargo-about` yet though.
